Src/lib/UL/spinner.c

```diff
@@ -39,22 +39,30 @@ static obj_led_t led_arr[] =
 };
 
 static spinner_state_t spinner_state = e_spinner_state_run;
+static uint32_t        period        = 400;
+static _Bool           is_timeout    = 0;
+static queue_t         queue;
```

Comment:
Looks not so good to be placed in this layer, any reason for that?

Reply:
Done